win32ss/gdi/eng/pdevobj.h

```diff
@@ -61,7 +61,7 @@ typedef struct _GRAPHICS_DEVICE
     struct _GRAPHICS_DEVICE * pNextGraphicsDevice;
     struct _GRAPHICS_DEVICE * pVgaDevice;
     PDEVICE_OBJECT   DeviceObject;
-    PVOID            pDeviceHandle;
+    PDEVICE_OBJECT   PhysDeviceHandle;
```

Comment:
device "Handle" sounds like it would have to be some sort of a handle, not a direct `PDEVICE_OBJECT` 🤔

Reply:
Yes, dunno why it's called like that. `DeviceObject` contains just DO, i.e. link to the video driver itself. VideoPrt previously returned pointer to the same DO object, so both `DeviceObject` and `PhysDeviceHandle` members were equal - which was incorrect and fixed :)